task1/Program.cs

```diff
@@ -45,5 +71,792 @@ public static decimal PerformCalculation(int r)
+        {
+            Console.WriteLine("Input sallary");
+            double salary = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Input stage");
+            int stage = Convert.ToInt32(Console.ReadLine());
+            double bonus = 0;
+
+            if (stage < 5)
+            {
+                bonus = 0.1;
+            }
+            if (stage >= 5 && stage < 10)
+            {
+                bonus = 0.15;
+            }
+            if (stage >= 10 && stage < 15)
+            {
+                bonus = 0.25;
+            }
+            if (stage >= 15 && stage < 20)
+            {
+                bonus = 0.35;
+            }
+            if (stage >= 20 && stage < 25)
+            {
+                bonus = 0.45;
+            }
+            if (stage >= 25)
+            {
+                bonus = 0.5;
+            }
+            double result = salary + bonus * salary;
+            Console.WriteLine($"Salary {salary} and stage {stage}, bonus is {bonus * salary} and result is {result}");
+        }
+
+        public static void Factorial()
+        {
+            Console.WriteLine("Input number");
+            int number = Convert.ToInt32(Console.ReadLine());
+            int result = 1;
+            for (int i = 1; i <= number; i++)
+            {
+                result *= i;
+            }
+            Console.WriteLine($"Factorial {number} is {result}");
+        }
+
+        public static void DrawFigures()
+        {
+            Console.WriteLine("Input number of symbols to draw string");
+            int numberForString = Convert.ToInt32(Console.ReadLine());
+            DrawString(numberForString);
+            Console.WriteLine("Input number of symbols to draw rectangle");
+            Console.WriteLine("Input height");
+            int heightRectangular = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Input width");
+            int widthRectangular = Convert.ToInt32(Console.ReadLine());
+            DrawRectangularFrame(heightRectangular, widthRectangular);
+            DrawRectangular(heightRectangular, widthRectangular);
+            Console.WriteLine("Input number of symbols to draw triangles");
+            int triangleSide = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Right triangle");
+            DrawRightTriangularFrame(triangleSide);
+            DrawRightTriangular(triangleSide);
+            Console.WriteLine("Equilateral triangle");
+            DrawEquilateralTriangleFrame(triangleSide);
+
+
+
+        }
+
+        private static void AnalyzeOfNumber()
+        {
+            Console.WriteLine("Input number to analyze");
+            int testNumber = Convert.ToInt32(Console.ReadLine());
+            //
+            // string to aggregate report of analyze.
+            //
+            string report = "";
+
+            //
+            // Checking if number is positive or negative.
+            //
+            if (testNumber > 0)
```

Comment:
testNumber variable can contain value that more OR less zero, not both of them.
So please use else if statement in these cases to avoid excessive calculations.